linera-client/src/wallet.rs

```diff
@@ -194,6 +198,7 @@ impl UserChain {
             timestamp,
             next_block_height: BlockHeight::ZERO,
             pending_proposal: None,
+            epoch: Epoch::ZERO,
```

Comment:
We pass in a timestamp here; do we also have the correct epoch at the call site?